rasa_nlu_examples/extractors/flashtext_entity_extractor.py

```diff
@@ -0,0 +1,137 @@
+import os
+from typing import List, Type, Optional, Dict, Text, Any
+
+import rasa.shared.utils.io
+
+from rasa.nlu.components import Component
+from rasa.nlu.config import RasaNLUModelConfig
+from rasa.nlu.extractors.extractor import EntityExtractor
+from rasa.nlu.extractors.regex_entity_extractor import RegexEntityExtractor
+from rasa.nlu.model import Metadata
+from rasa.nlu.tokenizers.tokenizer import Tokenizer
+from rasa.shared.nlu.constants import (
+    TEXT,
+    ENTITY_ATTRIBUTE_TYPE,
+    ENTITY_ATTRIBUTE_START,
+    ENTITY_ATTRIBUTE_END,
+    ENTITY_ATTRIBUTE_VALUE,
+    ENTITIES,
+)
+from rasa.shared.nlu.training_data.message import Message
+from rasa.shared.nlu.training_data.training_data import TrainingData
+
+from flashtext import KeywordProcessor
+
+
```

Comment:
Might be good to add documentation as well with an example config.yml. We're using `mkdocs` for this. You can run the docs locally via `mkdocs serve`.